server/odc-migrate/src/main/resources/migrate/common/V_4_2_1_6__alter_connect_database.sql

```diff
@@ -0,0 +1,4 @@
+alter table `connect_database` modify `charset_name` varchar(128) default null comment 'database charset name';
+alter table `connect_database` modify `collation_name` varchar(128) default null comment 'database collation name';
+alter table `connect_database` modify `table_count` bigint(20) default null comment 'table count which belongs to the database';
+alter table `connect_database` modify `last_sync_time` datetime not null default CURRENT_TIMESTAMP comment 'table count which belongs to the database';
```

Comment:
comment of field `last_sync_time` incorrect

Reply:
fixed